coffee-client/src/components/order.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from 'react';
+import styled, { css } from 'react-emotion';
+import { connect } from 'react-redux';
+import * as CoffeeActions from '../redux/actions/coffees';
+import { InputButton } from './buttons';
+import { Label, InputField, SelectInput, LabelSelect } from './input-fields';
+
+const Form = styled('form')`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: left;
+`;
+
+const Title = styled('div')`
+  margin-top: 5%;
+  margin-left: 10%;
+  font-weight: 400;
+  font-size: 1.9em;
+`;
+
+const Orderid = styled('div')`
+  margin-top: -1.5%;
+  margin-left: 17%;
+  font-weight: 400;
+  font-size: 1em;
+`;
+
+const Total = styled('div')`
+  margin-top: -1.8%;
+  margin-left: 51%;
+  font-weight: 400;
+  font-size: 1.5em;
+`;
+
+const Hr = styled ('hr') `
+  display: block;
+  border: 0;
+  width: 50%;
+  margin-top: 2%;
+  margin-left: 10%;
+  margin-tbottom: 2%;
+  border-top: 2px solid black;
+`;
+
+const CoffeeBox = styled('div')`
+  display: flex;
+  flex-direction: column;
+  overflow-x: scroll;
+  height: 17%;
+  width: 25%;
+  position: absolute;
+  letter-spacing: -0.5px;
+  top: 26.8%;
+  right: 40%;
+  background-color: transparent;
+  color: black;
+  text-align: right;
+`;
+
+
+class Order extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      kg: '',
+      total: '',
+      notes: ''
+    }
+  }
+
+  coffeeId = ':coffee_id';   // to change to passed url params later
+  // coffeeId = this.props.match.params.coffeeId
+
+  componentDidMount() {
+    this.props.getCoffee(this.coffeeId);
+  }
+
+  handleInput = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  // handleSubmit = event => {
+  //   event.preventDefault();
+  //   this.props.createTransaction(this.price);
+  //   this.props.history.push('order/:id_order');
+  // }
+
+  // create a function for the price 
+
+  id = 'ecedd2e7-c913-4250-a331-932c219c8000'    // to change later
+
+  render() {
+
+    const { coffees } = this.props;
+
+    // coffeeBox
+
+    let altitude = coffees[this.id] && coffees[this.id].altitude;
+    let bean_density = coffees[this.id] && coffees[this.id].bean_density;
+    let business_name = coffees[this.id] && coffees[this.id].Producer.business_name;
+    let botanical_variety = coffees[this.id] && coffees[this.id].botanical_variety;
+    let name = coffees[this.id] && coffees[this.id].name;
+    let roast_appearance = coffees[this.id] && coffees[this.id].roast_appearance;
+    let preparation = coffees[this.id] && coffees[this.id].preparation;
+    let rating = coffees[this.id] && coffees[this.id].rating;
```

Comment:
TOO
MANY
PUSH-UPS.

Reply:
Use `const` always. Then if you later find you have to change the value of a variable, come back and chang that `const` to `let`.